gameofthrones/app/controllers/characters_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CharactersController < ApplicationController
+  def index
+    @characters = Character.all
+  end
+
+  def show
+    @character = Character.find(params[:id])
+  end
+
+  def new
+    @character = Character.new
+    @houses = House.all.map do |house|
+      [house.name, house.id]
+    end
+  end
+
+  def create
+    character = Character.new(character_params)
+    character.house_id = params[:house_id]
+    character.save!
+    redirect_to character
```

Comment:
just a note: you may not want to pass a `!` on Active Record methods in a controller  method in production.